tests/scripts/test_workflow.py

```diff
@@ -161,3 +161,12 @@ def test_workflow_error_policy_fail_phase(repository):
         assert tuple(sorted([order.pop(0), order.pop(0)])) == ('AfterCommonThirdActor', 'AfterThirdActor')
         assert not order
         assert workflow.errors and len(workflow.errors) == 2
+
+
+def test_workflow_render_dialogs(repository):
+    workflow = repository.lookup_workflow('UnitTest')()
+    with tempfile.NamedTemporaryFile(mode='w') as stdin_dialog:
+        stdin_dialog.write('Yes\n')
+        stdin_dialog.seek(0)
+        with mock.patch('sys.stdin.fileno', return_value=stdin_dialog.fileno()):
+            workflow.run(skip_dialogs=False)
```

Comment:
Just for the others, the PR is ok, we are just trying to figure out how to check the stdout.